### ISSUE TITLE
Output in Python API

### ISSUE THREAD

**Issue by sumitsidana** [aksnzhy/xlearn#334]
Hello, I don't get any output during using Python API version of XLEARN. I do get a very nice output when I use command line API of XLEARN. Is this output not available for Python? 

`ffm_model.fit(param, "./model.out")`

This runs, but, doesn't print any epoch or metric values!

**Comment by ZJXlooo** [comment]
same here, anyone got solutions?
